Subject: [PATCH] m68k: ARAnyM virtual Ethernet missing include

Add missing #include <asm/virtconvert.h> (needed for virt_to_phys())

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 arch/m68k/emu/nfeth.c |    1 +
 1 file changed, 1 insertion(+)

--- linux-m68k-2.6.21.orig/arch/m68k/emu/nfeth.c
+++ linux-m68k-2.6.21/arch/m68k/emu/nfeth.c
@@ -14,6 +14,7 @@
 #include <linux/module.h>
 #include <net/ieee80211.h>
 #include <asm/natfeat.h>
+#include <asm/virtconvert.h>
 
 enum {
 	GET_VERSION = 0,	/* no parameters, return NFAPI_VERSION in d0 */
